test/python/ibmq/test_ibmqjob.py

```diff
@@ -258,7 +258,12 @@ def test_retrieve_job(self, qe_token, qe_url):
 
         qobj = compile(self._qc, backend)
         job = backend.run(qobj)
-        rjob = backend.retrieve_job(job.job_id())
+
+        try:
+            rjob = backend.retrieve_job(job.job_id())
+        except DeprecatedFormatJobError as err:
+            self.skipTest(err.message)
```

Comment:
Small nit, I'd make the argument this should be an expected failure state not a skip. But not a big deal worth blocking over.